packages/react/src/VideoPlayer/components/icon-control/IconControl.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import clsx from 'clsx'
+import {Tooltip} from '../index'
+
+import styles from '../../VideoPlayer.module.css'
+
+type IconControlProps = {
+  tooltip: string
+} & React.HTMLAttributes<HTMLButtonElement>
+
+export const IconControl = ({tooltip, children, className, ...rest}: IconControlProps) => {
+  return (
+    <button className={clsx(styles.VideoPlayer__iconControl, className)} {...rest} aria-label={tooltip}>
+      {children}
+      <span className={styles.VideoPlayer__srOnly}>{tooltip}</span>
```

Comment:
Can we use our `visually-hidden` class here instead of creating a new style for it?

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5